README.md

```diff
@@ -11,11 +11,16 @@ AI-powered natural language end-to-end testing framework.
 
 ## Installation
 ```bash
-npm install @antiwork/shortest
+npm install -D @antiwork/shortest
 # or
-pnpm add @antiwork/shortest
+pnpm add -D @antiwork/shortest
 # or
-yarn add @antiwork/shortest
+yarn add -D @antiwork/shortest
+```
+
+Add `.shortest/` to your `.gitignore`:
```

Comment:
why?

Reply:
`-D` to add shortest as a dev dependency because it is not needed when bundling an app. 

The other one is just to let the users know that .shortest/ is not needed to be tracked.